api/grpcserver/config.go

```diff
@@ -35,6 +35,7 @@ const (
 	Activation               Service = "activation"
 	Smesher                  Service = "smesher"
 	Post                     Service = "post"
+	PostInfo                 Service = "postinfo"
```

Comment:
NIT: to be in line with the naming for the logger:

```suggestion
	PostInfo                 Service = "postInfo"
```